deeprankcore/query.py

```diff
@@ -212,7 +220,8 @@ def process( # pylint: disable=too-many-arguments, too-many-locals
         cpu_count: Optional[int] = None,
         combine_output: Optional[bool] = True,
         grid_settings: Optional[GridSettings] = None,
-        grid_map_method: Optional[MapMethod] = None
+        grid_map_method: Optional[MapMethod] = None,
+        grid_augmentation_count: Optional[int] = 0
```

Comment:
Optional[X] is equivalent to X | None (or Union[X, None]).

Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the `Optional` qualifier on its type annotation just because it is optional. 

So either we set the default to `None` (modifying the code accordingly) or we keep 0 removing the `Optional` hint.